Mathlib/Order/Filter/Lift.lean

```diff
@@ -5,7 +5,6 @@ Authors: Johannes Hölzl
 -/
 import Mathlib.Order.Filter.Prod
 import Mathlib.Order.ConditionallyCompleteLattice.Basic
-import Mathlib.Order.Filter.Finite
```

Comment:
Could we have an `assert_not_exists` here (or even in some significant file downstream of this where it is still true?) guarding against regression?